bedrock/mozorg/templates/mozorg/home/home-de.html

```diff
@@ -1,186 +0,0 @@
-{#
- This Source Code Form is subject to the terms of the Mozilla Public
- License, v. 2.0. If a copy of the MPL was not distributed with this
- file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-{% from "macros-protocol.html" import split, billboard, card, call_out_compact, content_card with context %}
-{% from "mozorg/home/includes/macros.html" import download_banner, download_banner_secondary, fxa_banner %}
-
-{% extends "mozorg/home/base.html" %}
-
-{% block page_title %}Internet für Menschen, nicht für Profit{% endblock %}
-{% block page_desc %}Mozilla ist die gemeinnützige Organisation hinter Firefox. Menschen sollten auch online die Kontrolle über ihr Leben haben. Dafür setzen wir uns ein.{% endblock %}
-
-{% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
-{% set show_fundraising_banner = switch('fundraising-banner-giving2022') and ftl_has_messages('banner-fundraising-title', 'banner-fundraising-body') %}
-
-{% block page_css %}
-  {{ css_bundle('protocol-split') }}
-  {{ css_bundle('protocol-card') }}
-  {{ css_bundle('protocol-newsletter') }}
-  {{ css_bundle('protocol-call-out') }}
-  {{ css_bundle('home-eu') }}
```

Comment:
Can this `eu-home` bundle be removed? (looks like it's still hanging out in static-bundles.json)